### ISSUE TITLE
Tell something about associations' fetching (.populate())

### ISSUE THREAD

**Issue by lebarde** [balderdashy/sails-docs#314]
Hello,
Coming from [this issue](https://github.com/balderdashy/sails/issues/2308#issue-45581062), I want to open an issue on the documentation, as I assume there is a lack inside. This information is not obvious. For instance I remember that Symfony2 ORM fetches associations automatically, so this needs some additions.

## Query page
On the [Waterline query language page](http://sailsjs.org/#/documentation/concepts/ORM/Querylanguage.html), we should add a note in the bottom of the [Query language basics](http://sailsjs.org/#/documentation/concepts/ORM/Querylanguage.html?q=query-language-basics)' section, to add nomething like:

> **Note:** When fetching objects from entities, you don't get also the associations by default. To fetch these, use the `.populate()` method. See [Reference/Waterline/Populated Values](http://sailsjs.org/#/documentation/reference/waterline/populated-values) for more information.

## Association's page
This page is a bit empty. We should add semething like:

> **Note:** Waterline doesn't fetch the associations by default. In addition to the following pages, you may want to see [Reference/Waterline/Populated values](http://sailsjs.org/#/documentation/reference/waterline/populated-values) to learn how to fetch data from associations.

Regards,

**Comment by rachaelshaw** [comment]
@lebarde would you be down to write a PR for this?

**Comment by sailsbot** [comment]
Thanks for posting, @lebarde.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
